.github/workflows/test-and-check.yml

```diff
@@ -120,7 +120,7 @@ jobs:
           - os: ubuntu-24.04-arm
             os_name: v22, Linux
             node_version: 22
-            filter: '--filter="./packages/*"'
+            filter: '--filter="./packages/*" --filter="!./packages/kv-asset-handler"'
```

Comment:
Do we have a plan for this when we get to Node 22?

Reply:
Yes, we'll need to migrate the `kv-asset-handler` tests to use the Vitest integration (right now they run in Node with some global scope mocking)